utility/mm2_crawler

```diff
@@ -1067,8 +1063,11 @@ def worker(options, config, host):
         rc = 3
 
     logger.info("Ending crawl of %r with status %r" % (host, rc))
+    host.last_crawled = datetime.datetime.utcnow()
     logger.removeHandler(fh)
     fh.close()
+    session.commit()
```

Comment:
Are you sure we want to commit even if an error occurred?  Typically, you don't want to do this as it might mean the database records are in some kind of inconsistent or unexpected (exceptional) state.